lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
I'm now confused whether we are talking about `$propdoc` format for ES6 :sweat_smile: 

Either way - correct me if I'm wrong - `expression: "[AST_Node] description"` means `expression` is a reference to `AST_Node`, not an array of `AST_Node`s.

To denote an array of something an asterisk is used, e.g. https://github.com/mishoo/UglifyJS2/blob/3ca902258c24209699f0b5bd5b9654252e492272/lib/ast.js#L658